### PR TITLE
Added MobilePay Address support to mobilepay template

### PR DESCRIPTION
**Pull request by twarberg**

### CODE REVIEW

File: templates/form/mobilepay.liquid [hunk 1]
```diff
@@ -1,10 +1,16 @@
 <div class="container">
   {% include 'form/error' %}
   <form action="{{ model.redirect_url }}" method="POST" target="_top" class="autosubmit">
-    <input type="hidden" name="SessionToken" value="{{ model.session_token }}" />
+    <input type="hidden" name="{{ model.token_input_name }}" value="{{ model.session_token }}" />
     {% if true %}
```
Comment (ta) [inline templates/form/mobilepay.liquid]: Does this if-statement serve a purpose?

File: templates/form/mobilepay.liquid [hunk 2]
```diff
@@ -1,10 +1,16 @@
 <div class="container">
   {% include 'form/error' %}
   <form action="{{ model.redirect_url }}" method="POST" target="_top" class="autosubmit">
-    <input type="hidden" name="SessionToken" value="{{ model.session_token }}" />
+    <input type="hidden" name="{{ model.token_input_name }}" value="{{ model.session_token }}" />
```
Comment (ta) [inline templates/form/mobilepay.liquid]: Will this have an impact on merchants using the current or a previous release?

Comment (twarberg) [inline templates/form/mobilepay.liquid]: Value variable hasn't changed and hence it should have no impact on current nor previous releases. Use of addresses feature will require updating any custom branding with above change.

(Tested locally with current branding without issues)

File: templates/form/mobilepay.liquid (hunk 1 above)
Comment (twarberg) [inline templates/form/mobilepay.liquid]: not really :-)

**Pull request merged**